src/server/routes/entity/series.js

```diff
@@ -0,0 +1,241 @@
+		const {markup, props} = entityEditorMarkup(generateEntityProps(
+			'series', req, res, {}
+		));
+
+		return res.send(target({
+			markup,
+			props: escapeProps(props),
+			script: '/js/entity-editor.js',
+			title: props.heading
+		}));
+	}
+);
+
+
+router.post('/create/handler', auth.isAuthenticatedForHandler,
+	createOrEditHandler);
+
+
+/* If the route specifies a BBID, make sure it does not redirect to another bbid then load the corresponding entity */
+router.param(
+	'bbid',
+	middleware.redirectedBbid
+);
+router.param(
+	'bbid',
+	middleware.makeEntityLoader(
+		'Series',
+		[
+			'defaultAlias',
+			'disambiguation',
+			'seriesOrderingType',
+			'identifierSet.identifiers.type'
+		],
+		'Series not found'
+	)
+);
+
+function _setSeriesTitle(res) {
+	res.locals.title = utils.createEntityPageTitle(
+		res.locals.entity,
+		'Series',
+		utils.template`Series “${'name'}”`
+	);
+}
+
+router.get('/:bbid', middleware.loadEntityRelationships, middleware.loadEntities, (req, res) => {
+	_setSeriesTitle(res);
+	entityRoutes.displayEntity(req, res);
+});
+
+
+function seriesToFormState(series) {
+	const aliases = series.aliasSet ?
+		series.aliasSet.aliases.map(({languageId, ...rest}) => ({
+			...rest,
+			language: languageId
+		})) : [];
+
+	const defaultAliasIndex = entityRoutes.getDefaultAliasIndex(series.aliasSet);
+	const defaultAliasList = aliases.splice(defaultAliasIndex, 1);
+
+	const aliasEditor = {};
+	aliases.forEach((alias) => { aliasEditor[alias.id] = alias; });
+
+	const buttonBar = {
+		aliasEditorVisible: false,
+		identifierEditorVisible: false
+	};
+
+	const nameSection = _.isEmpty(defaultAliasList) ? {
+		language: null,
+		name: '',
+		sortName: ''
+	} : defaultAliasList[0];
+	nameSection.disambiguation =
+		series.disambiguation && series.disambiguation.comment;
+
+	const identifiers = series.identifierSet ?
+		series.identifierSet.identifiers.map(({type, ...rest}) => ({
+			type: type.id,
+			...rest
+		})) : [];
+
+	const identifierEditor = {};
+	identifiers.forEach(
+		(identifier) => { identifierEditor[identifier.id] = identifier; }
+	);
+	const seriesSection = {
+		orderType: series.seriesOrderingType && series.seriesOrderingType.id,
+		seriesType: series.entityType
+	};
+
+	const relationshipSection = {
+		canEdit: true,
+		lastRelationships: null,
+		relationshipEditorProps: null,
+		relationshipEditorVisible: false,
+		relationships: {}
+	};
+	series.relationships.forEach((relationship) => {
+		relationship.attributeSet.relationshipAttributes.forEach(attribute => {
+			relationship[`${attribute.type.name}`] = attribute.value.textValue || -Infinity;
+		});
+	});
+	/* eslint-disable no-nested-ternary */
+	if (series.seriesOrderingType.label === 'Manual') {
+		series.relationships.sort((a, b) => (a.position > b.position ? 1 : b.position > a.position ? -1 : 0));
+	}
+	else {
+		series.relationships.sort((a, b) => (a.number > b.number ? 1 : b.number > a.number ? -1 : 0));
+	}
```

Comment:
This code is also in another file, isn't it? If so we should extract it as a separate util, considering it's pretty dense code that we only want to change in one place if we ever need to change it